streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/StreamsBuilder.scala

```diff
@@ -176,10 +176,10 @@ class StreamsBuilder(inner: StreamsBuilderJ = new StreamsBuilderJ) {
    *
    * @see `org.apache.kafka.streams.StreamsBuilder#addGlobalStore`
    */
-  def addGlobalStore(storeBuilder: StoreBuilder[_ <: StateStore],
-                     topic: String,
-                     consumed: Consumed[_, _],
-                     stateUpdateSupplier: ProcessorSupplier[_, _]): StreamsBuilderJ =
+  def addGlobalStore[K, V](storeBuilder: StoreBuilder[KeyValueStore[K, V]],
```

Comment:
As I don't speak Scala, I am not sure if this change is correct -- it compiles so I was happy :)